src/test/functional/features/list-contact-types.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Contact Types List
```

Comment:
Question (not specifically a bad thing or a good thing, but a should we sort of thing): should we have a test to check that if we are not a super user, that we are not able to access the page

Is this covered elsewhere? Probably a unit test thing?

Reply:
I am sure that is covered in the api side testing